certified-connectors/ShortURL/apiProperties.json

```diff
@@ -4,6 +4,8 @@
       "actions"
     ],
     "connectionParameters": {},
-    "iconBrandColor": "#124f4f"
+    "iconBrandColor": "#232323",
+	"publisher":"APPS 365 LTD",
```

Comment:
Hi @cirrussoft, thank you for submitting an update for ShortURL. The artifacts should be well aligned. Please fix the alignment in the file. Thank you very much.